### PR TITLE
workaround for get_wa in calc

### PR DESCRIPTION
**Pull request by abjonnes**

### CODE REVIEW

File: modules/calc.py [hunk 1]
```diff
@@ -217,8 +217,13 @@ def get_wa(search):
...
     query = txt
     uri = 'https://tumbolia.appspot.com/wa/'
-    uri += urllib.quote(query.replace('+', '%2B'))
-    answer = web.get(uri)
+    # tumbolia appspot WA has bug where querystrings are decoded twice,
+    # and encoded plusses are changed to spaces
+    # e.g. '5%2B5' --> '5+5' --> '5 5'
+    # until this is fixed, encode the querystring twice
+    q = urllib.quote(query.replace('+', '%2B'))
+    q = urllib.quote(q)
```
Comment (kaneda) [inline modules/calc.py]: Interesting. I think this should be resolved upstream in Tumbolia @myano

Comment (abjonnes) [inline modules/calc.py]: i submitted a pull request there too but it hasn't been touched in 4 years

Comment (kaneda) [inline modules/calc.py]: @abjonnes @myano became the maintainer for Tumbolia fairly recently. I'll be sure to bring this up.

**Comment by myano** [comment]
This has been fixed on tumbolia (and tumbolia-two).

**Pull request closed without merging**